ros/src/waypoint_updater/cfg/DynReconf.cfg

```diff
@@ -17,9 +17,9 @@ gen.add("dyn_default_velocity", double_t, 0, "top speed to use (mps)",
 gen.add("dyn_default_accel",    double_t, 0, "accel/decel rate (m/s) to use",
         0.8, 0.5, 5.0)
 gen.add("dyn_tl_buffer",        double_t, 0, "buffer distance (m) to stop at before traffic light wp",
-        3.5, 0.0, 5.0)
-gen.add("dyn_buffer_offset",    double_t, 0, "offset into buffer to target stopping point",
         2.5, 0.0, 5.0)
+gen.add("dyn_buffer_offset",    double_t, 0, "offset into buffer to target stopping point",
+        1.0, 0.0, 5.0)
 gen.add("dyn_creep_zone",       double_t, 0, "zone before stoplight where no speedup beyond creep",
```

Comment:
Actually, I thought this change was to target the stopping point closer to the line, but actually it shifts from (3.5 - 2.5) = 1.0 m before line to (2.5 - 1.0) = 1.5 m before the line, so isn't it a little farther away?

Reply:
Also, reducing the tl_buffer increases the chance that it could get stuck in the slowdown state when DBW manual braking activates.  I got this a couple times during testing short speedup/slowdown cycles.  Did you see any issues with this?

Reply:
Also changing the DBW manual braking threshold `self.manual_braking_upper_velocity_limit` from 1.5 m/s to 1.4 m/s seems to fix this so it doesn't get stuck in slowdown.  Can you add this change to this PR?

Reply:
Ok cool, I’ll try it out later tonight.